pkg/bindings/containers/types.go

```diff
@@ -176,7 +176,7 @@ type UnpauseOptions struct{}
 //go:generate go run ../generator/generator.go WaitOptions
 // WaitOptions are optional options for waiting on containers
 type WaitOptions struct {
-	Condition *define.ContainerStatus
+	Condition []define.ContainerStatus
```

Comment:
I deliberately choose not to change this to plural for preserving some degree of compatibility.
With this older client can call new server safely. Similarly new client can call older server, but only as long as the contains at most one element.

Reply:
If I understand generator correctly uri param is derived just from member name. Maybe we can introduce some custom  tag. So it could look like:
```go
type WaitOptions struct {
	Conditions []define.ContainerStatus `uriparam:"condtion"`
	Interval   *string                  `uriparam:"interval"`
}
```
Meaning the name is changed, but on wire it's the same.